sdk/storage/Azure.Storage.Webjobs.Extensions.Common/tests/WebJobsTestEnvironment.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core.TestFramework;
+
+namespace Azure.WebJobs.Extensions.Storage.Common.Tests
+{
+    public class WebJobsTestEnvironment : TestEnvironment
+    {
+        public WebJobsTestEnvironment()
+        : base("storage") { }
+
+        public string PrimaryStorageAccountConnectionString => GetVariable("AzureWebJobsStorage".ToUpperInvariant());
```

Comment:
doesn't it work without `ToUpperInvariant`?

Reply:
It doesn't. See https://dev.azure.com/azure-sdk/internal/_build/results?buildId=563852&view=results . Here's what happens:
- arm template/pwsh/azure devops combo capitalize all env variables coming out from test resources generation, (there are some hacks to force it do otherwise but would complicate these scripts)
- Environment.GetEnvironmentVariable's parameter is case insensitive on Windows but it's sensitive in Linux/OS-X :/
- IConfiguration and it's family on the other hand seems to be case insensitive anywhere...

Reply:
Should we just capitalize the string constants?